smartsim/_core/entrypoints/service.py

```diff
@@ -42,19 +42,21 @@ class Service(ABC):
     def __init__(
```

Comment:
In the Class doc string, should Base contract be base class?

Reply:
Abstract base class*

Reply:
rephrasing to improve clarity of class doc string